crowbar_framework/app/helpers/installer/upgrades_helper.rb

```diff
@@ -69,20 +79,28 @@ def check_ceph_repo?
       true
     end
 
-    def check_repos?
-      check_ha_repo? != false && check_ceph_repo? != false
+    def upgrade_repos_present?
+      if upgrade_ha_repo_needed?
+        return false unless upgrade_ha_repo?
+      end
+
+      if upgrade_ceph_repo_needed?
+        return false unless upgrade_ceph_repo?
+      end
     end
 
     def restored?
       Crowbar::Backup::Restore.status[:success]
     end
 
-    def database_node
-      @node ||= NodeObject.find("crowbar_upgrade_db_dumped_here:true").first
+    def upgrade_database_backup
+      @node ||= NodeObject.find(
+        "crowbar_upgrade_db_dumped_here:true"
+      ).first
     end
 
-    def database_backup_path
-      database_node[:crowbar][:upgrade][:db_dump_location]
+    def upgrade_database_backup
```

Comment:
Lint/DuplicateMethods: Method `Installer::UpgradesHelper#upgrade_database_backup` is defined at both /app/crowbar_framework/app/helpers/installer/upgrades_helper.rb:96 and /app/crowbar_framework/app/helpers/installer/upgrades_helper.rb:102.